packages/replay-next/src/hooks/useNoSelectContentEditable.tsx

```diff
@@ -0,0 +1,13 @@
+import { CSSProperties, MutableRefObject, ReactNode, use, useLayoutEffect, useRef } from "react";
+
+export function useNoSelectContentEditable({
+  editable,
+  ref,
+}: {
+  editable: boolean;
+  ref: MutableRefObject<HTMLElement | null>;
+}) {
+  useLayoutEffect(() => {
+    // TODO
```

Comment:
:hand: missing something here?

Reply:
Haha, I didn't mean to `git add` this file. It was part of the follow up. Will delete it.